File: 3st/Hyeonseo/programmer/핸드폰 번호 가리기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string phone_number) {
    string answer = "";
    int n = phone_number.length();
    
    for (int i = 0; i < n; i++) {
        if (i < n - 4) {
            answer += "*";
        } else {
            answer += phone_number[i];
        }
    }
    
    return answer;
}

File: 3st/Hajeong/programmer/핸드폰 번호 가리기.java
class Solution {
    public String solution(String phone_number) {
        String answer = "";
        char[] num = phone_number.toCharArray();

        for (int i = 0; i < num.length - 4; i++) {
            answer += '*';
        }
        for (int i = num.length - 4; i < num.length; i++) {
            answer += num[i];
        }
        return answer;
    }
}